apps/projects/app/components/Content/Settings.js

```diff
@@ -421,28 +437,24 @@ Settings.propTypes = {
   onLogin: PropTypes.func.isRequired,
 }
 
-const StyledInputDropDown = ({ children }) => {
+const StyledInputDropDown = ({ children, className }) => {
```

Comment:
Could you please elaborate on why we need to use the `className` prop here?

Reply:
I spent some hours refreshing my knowledge about styled-components , so now I caught on it. Sorry for the inconvenience Chad.